docs/_documentations/troubleshooting.md

```diff
@@ -1010,3 +1011,20 @@ Install and enable the [Java Extension Pack](https://marketplace.visualstudio.co
 
 ### Debugger fails to attach after restarting project into Debug mode
 Run the `attach debugger` command again. If the issue persists after a few attempts, restart the project in **Debug** mode a second time.
+
+***
+# Restoring lost connections
+
+<!--
+Action/Topic: Restoring lost connections
+Issue type: bug
+Issue link: https://github.com/eclipse/codewind/issues/3118
+Codewind version: 0.13.0
+-->
+## Node.js remote debugging disconnects
```

Comment:
This will likely affect more than just Node.js since it is an issue with the port-forwarding which is used for Java debug sessions as well.  Change to "Remote debugging disconnects" and below to "The debug connection might be lost when debugging projects remotely."